dom-manipulation/main.js

```diff
@@ -0,0 +1,26 @@
+let numOfClicks = 0;
+const $buttonHot = document.querySelector('.hot-button');
+const $clickCount = document.querySelector('.click-count');
+
+function incrementClick() {
+  numOfClicks++;
+  $clickCount.textContent = 'Clicks: ' + numOfClicks;
+}
+$buttonHot.addEventListener('click', incrementClick);
+
+function updateClassName(event) {
+  if (numOfClicks < 4) {
+    event.target.className = 'hot-button cold';
+  } else if (numOfClicks < 7) {
+    event.target.className = 'hot-button cool';
+  } else if (numOfClicks < 10) {
+    event.target.className = 'hot-button tepid';
+  } else if (numOfClicks < 13) {
+    event.target.className = 'hot-button warm';
+  } else if (numOfClicks < 16) {
+    event.target.className = 'hot-button hot';
+  } else {
+    event.target.className = 'hot-button nuclear';
+  }
+}
+$buttonHot.addEventListener('click', updateClassName);
```

Comment:
You should only have 1 event listener on your `$buttonHot`. You don't want to keep adding the same type of event over and over to an element. An element should only ever have 1 type of event attached to it at a time. So an element can have a `click` event and a `dblclick` event, but it shouldn't have 2 separate `click` events, 

So a better approach would be to create a `handleClick` function, and use that as your callback function, and then inside of `handleClick`, you can call `incrementClick` & `updateClassName`